src/arch/sparcv9/CodeGen.zig

```diff
@@ -771,8 +771,7 @@ fn airArg(self: *Self, inst: Air.Inst.Index) !void {
     // TODO Copy registers to the stack
     const mcv = result;
 
-    // TODO generate debug info
-    // try self.genArgDbgInfo(inst, mcv);
+    try self.genArgDbgInfo(inst, mcv, @intCast(u32, arg_index));
```

Comment:
This TODO is introduced in the previous commit and removed right afterwards.

Reply:
So, about this one, originally I realized that the dbg_arg instruction is currently useless so I removed it and replaced the emission part with that TODO.
However, a while later I took a read at other backends and found that the RISC-V backend has a working implementation, so I wrote a `genArgDbgInfo` based on that impl and replaced that TODO with a call to the real function.

Is there anything I should do about it?

Reply:
See my comment on the first review part. Commits should be atomic and shouldn't introduce lines in one commit and remove it in the next within the same PR.